opensrp-app/src/main/java/org/smartregister/sync/ClientProcessorForJava.java

```diff
@@ -941,6 +942,17 @@ private Field getField(Class clazz, String fieldName) {
         return getField(clazz.getSuperclass(), fieldName);
     }
 
+    /**
+     * Update the fts table with the provided values. This overloaded method adds the parameter entityType to uniquely distiguish the client types being processed
+     *
+     * @param tableName     the case table
+     * @param entityType    the client type or bind type
+     * @param entityId      the entity identifier
+     * @param contentValues the fields to update and corresponding values
+     */
+    public void updateFTSsearch(String tableName, String entityType, String entityId, ContentValues contentValues) {
```

Comment:
When is the entityType parameter used? Is it a placeholder for now?

Reply:
Yeah on this library it is a place holder, however most implementations override this method with their own custom implementation. Whilst this implementation maintains backward compatibility, it also provides the entity type to implementations requiring it in the processing.